### PR TITLE
Timeboxed code coverage

### PR DESCRIPTION
**Pull request by omnivion**
### Information
- JIRA story QPPCT-570 and QPPCT-561

### Changes proposed in this PR:
- Timeboxed code coverage improvements
- Standard way of testing our enums

### Checklist 
- [X] All JUnit tests pass (`mvn clean verify`).
- [X] New unit tests written to cover new functionality.
- [X] Added and updated JavaDocs for non-test classes and methods.
- [X] No local design debt. Do you feel that something is "ugly" after your changes?
- [X] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

File: converter/src/test/java/gov/cms/qpp/conversion/ConverterTest.java [hunk 1]
```diff
@@ -20,28 +53,6 @@
...
-import static com.google.common.truth.Truth.assertWithMessage;
-import static org.junit.Assert.fail;
-import static org.mockito.ArgumentMatchers.any;
-import static org.powermock.api.mockito.PowerMockito.doThrow;
-import static org.powermock.api.mockito.PowerMockito.mock;
-import static org.powermock.api.mockito.PowerMockito.mockStatic;
-import static org.powermock.api.mockito.PowerMockito.when;
-import static org.powermock.api.mockito.PowerMockito.whenNew;
 
 @RunWith(PowerMockRunner.class)
```
Comment (clydet) [inline converter/src/test/java/gov/cms/qpp/conversion/ConverterTest.java]: How are things going with the power mock extension?

File: converter/src/test/java/gov/cms/qpp/conversion/validate/QrdaValidatorTest.java [hunk 2]
```diff
@@ -10,6 +10,7 @@
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.powermock.api.mockito.expectation.PowerMockitoStubber;
```
Comment (clydet) [inline converter/src/test/java/gov/cms/qpp/conversion/validate/QrdaValidatorTest.java]: Unused import?

**Comment by sfradkin** [comment]
@omnivion: Check @clydet's comments and then this is good to merge.

File: converter/src/main/java/gov/cms/qpp/conversion/xml/XmlUtils.java [hunk 3]
```diff
@@ -3,6 +3,8 @@
 
 import java.io.UncheckedIOException;
 import java.nio.charset.Charset;
```
Comment (FlexionSonarQube) [inline converter/src/main/java/gov/cms/qpp/conversion/xml/XmlUtils.java]: MINOR Remove this unused import 'java.nio.charset.Charset'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: rest-api/src/main/java/gov/cms/qpp/conversion/api/services/ValidationServiceImpl.java [hunk 4]
```diff
@@ -25,6 +25,7 @@
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.nio.charset.Charset;
```
Comment (FlexionSonarQube) [inline rest-api/src/main/java/gov/cms/qpp/conversion/api/services/ValidationServiceImpl.java]: MINOR Remove this unused import 'java.nio.charset.Charset'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java [hunk 5]
```diff
@@ -1,6 +1,8 @@
 package gov.cms.qpp.conversion.validate;
 
 import com.google.common.base.Strings;
+import com.google.common.collect.Collections2;
```
Comment (FlexionSonarQube) [inline converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java]: MINOR Remove this unused import 'com.google.common.collect.Collections2'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by FlexionSonarQube** [comment]
SonarQube analysis reported 3 issues
* ![MINOR][MINOR] 3 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: converter/src/test/java/gov/cms/qpp/conversion/ConverterTest.java (hunk 1 above)
Comment (omnivion) [reply converter/src/test/java/gov/cms/qpp/conversion/ConverterTest.java]: different PR

File: converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java [hunk 6]
```diff
@@ -1,6 +1,7 @@
 package gov.cms.qpp.conversion.validate;
 
 import com.google.common.base.Strings;
+
```
Comment (FlexionSonarQube) [inline converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java]: MINOR Remove this unused import 'com.google.common.collect.Collections2'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by FlexionSonarQube** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java (hunk 6 above)
Comment (omnivion) [reply converter/src/main/java/gov/cms/qpp/conversion/validate/Checker.java]: IT IS REMOVED YOU MADMAN

**Pull request merged**